doc/articles/features/svg.md

```diff
@@ -10,6 +10,24 @@ Uno Platform supports using vector SVG graphics inside of your cross-platform ap
 
 ## How to use SVG
```

Comment:
fyi @jeromelaban, maybe best to sync with @vatsashah45 and @dansiegel regarding the current PR that was started here https://github.com/unoplatform/uno/pull/16301#issuecomment-2057961771

In addition to the latest details for svg:
- https://github.com/unoplatform/uno/pull/16300
- https://github.com/unoplatform/uno-private/issues/455

Reply:
Yes that was the intent, I merged the content from the other PR. It will close as this PR merges.